dist/bundler.js

```diff
@@ -119,14 +98,8 @@ function removeExistingSourceMap(outfile) {
   }
 }
 
-function logBuild(outFile, opts) {
-  var resolution = opts.lowResSourceMaps ? '' : 'high-res ';
-  _jspmLibUi2['default'].log('ok', 'Built into `' + outFile + '`' + (opts.sourceMaps ? ' with ' + resolution + 'source maps' : '') + ', ' + (opts.minify ? '' : 'un') + 'minified' + (opts.minify ? (opts.mangle ? ', ' : ', un') + 'mangled.' : '.'));
-}
-
 function getFullModuleName(moduleName, map) {
   var matches = [];
-
   _Object$keys(map).forEach(function (m) {
     if (m.startsWith(moduleName)) {
```

Comment:
With this check bundling 'moment' and 'moment-datepicker' throws 'Version conflict...' error.